fs-one-concepts.md

```diff
@@ -4,18 +4,54 @@ Try your best to answer each question on your own before looking up the answer o
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
 
+Enzyme, like Jest for JavaScript, is a way to test for React. 
+
+Google: Enzyme is a JavaScript Testing utility for React that makes it easier to assert, manipulate, and traverse your React Components’ output. Methods: .toEqual(), .text(), .simulate()
+
 #### 2. What is the difference between dynamic and a static routes?
 
+Google: In the Static routing, the table is set up and modified manually whereas in the Dynamic routing the table is built automatically with the help of the routing protocols.
+
 #### 3. What is a JSON API?
 
+A JSON API returns JSON objects to be rendered on the page.
+
+Google: JSON API documents work as any other API format; you send a request to an endpoint and receive your document. The JSON API specification defines how resources are structured. This structure helps in normalizing how you consume the API. Requests will be sent and received as JSON objects.
+
 #### 4. What is a migration and why would you use one?
 
+Migrations allow you to make updates to your database. You can add columns, destroy rows, etc. through migrations. Once you complete your changes in the migration file, make sure to save, then db:migrate again to set the changes in the database.
+
+Google: Requirements change all the time, and those changes often lead to database changes. Migrations give you a way to modify your database schema within your Rails application. So you use Ruby code instead of SQL. Using Rails migrations instead of SQL has several advantages. Rails migrations are useful any time you need to make a change to your application’s database.
+
 #### 5. Explain how to set up a route in React.
 
+npm install react-router-dom
+import ReactDOM from 'react-dom,
+import { Route, Link, BrowserRouter as Router } from 'react-router-dom
+
+get '/main' => 'main#index'
```

Comment:
Great so far! This get however is an example of a Rails route. What does a route look like in React?